src/core/options-watcher.h

```diff
@@ -59,6 +59,7 @@ class options_watcher
     std::thread _updater;
     std::mutex _mutex;
     std::condition_variable _stopping;
+    std::atomic_bool _alive = true;
```

Comment:
If there's a ctor, please put this there.

Reply:
Moved